test/integration/basic/controls/gcloud.rb

```diff
@@ -0,0 +1,68 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project_id = attribute('project_id')
+region = attribute('region')
+router = attribute('router_name')
+nat = attribute('name')
+
+control "gcloud" do
+  title "Google Compute Router NAT configuration"
+  describe command("gcloud compute routers nats describe #{nat} --router #{router} --region #{region} --format json") do
+    its(:exit_status) { should eq 0 }
+    its(:stderr) { should eq '' }
+
+    let!(:data) do
+      if subject.exit_status == 0
+        JSON.parse(subject.stdout)
+      else
+        {}
+      end
+    end
+
+    describe "nat" do
+      it "generated a NAT name" do
```

Comment:
See above about asserting positively

Reply:
In this example its testing that the module itself (not the fixture or example) generated a name.